gradle/build-dist.gradle

```diff
@@ -50,10 +50,19 @@ task copyZookeeperInstall(type: Copy) {
 }
 
 task copyXDInstall(type: Copy, dependsOn: [":spring-xd-dirt:build", ":spring-xd-dirt:installApp"]) {
+	def versions = [:]
+	def file = rootProject.file('dependencies.properties')
+		file.withReader { reader ->
+			def properties = new Properties()
+			properties.load(reader)
+			versions << properties
+		}
+	def version = versions["com.google.protobuf:protobuf-java"]
```

Comment:
This logic is starting to get crazy.
In any case, if we go through with this, need to rename `version` to something more descriptive (_e.g_ `protobufVersion`)